docs/deployment.md

```diff
@@ -0,0 +1,41 @@
+# ESI Deployment
+
+ESI is composed of multiple OpenStack services. As a result, any OpenStack deployment tool that can deploy and configure the required services can create an ESI deployment. Our recommended deployment tool is [Standalone TripleO](https://docs.openstack.org/project-deploy-guide/tripleo-docs/latest/deployment/standalone.html). Standalone TripleO is part of the active upstream [TripleO project](https://docs.openstack.org/tripleo-docs/latest/), and is highly configurable.
+
+If you decide to use Standalone TripleO, you will need the following additional configurations before deployment.
+
+## Ironic Configuration
+
+Standalone TripleO requires the following configuration changes in order to deploy Ironic. Note that Ironic works better with openvswitch.
```

Comment:
This note about openvswitch seems out of place (because you haven't suggested that there is an alternative). I would just drop it...

Reply:
Done!